### PR TITLE
Feature/UI 21

### PR DESCRIPTION
**Pull request by lmiller1990**
Notice the `...` for the long name. 

image

### CODE REVIEW

File: src/SmartList/SmartList.stories.tsx [hunk 1]
```diff
@@ -63,7 +62,34 @@ const content: IRowElement[] = [
...
+    columns: [
+      {
+        name: 'id',
+        element: <div>3</div>,
+        text: '3',
+      },
+      {
+        name: 'email',
+        element: (
+          <div style={{ textOverflow: 'ellipsis' }}>
```
Comment (MichaelDorkhom) [inline src/SmartList/SmartList.stories.tsx:85]: Do you need `style={{ textOverflow: 'ellipsis' }}` if it's been set in css?

Comment (lmiller1990) [reply src/SmartList/SmartList.stories.tsx:85]: good question

let me check that out

**Comment by julianjuko** [comment]
address Michael's comment first though

File: src/SmartList/SmartList.stories.tsx (hunk 1 above)
Comment (lmiller1990) [reply src/SmartList/SmartList.stories.tsx:85]: its not. removed

**Pull request merged**